sdk/search/Azure.Search.Documents/src/Generated/Models/SearchOptions.cs

```diff
@@ -21,67 +21,6 @@ public SearchOptions()
             ScoringParameters = new ChangeTrackingList<string>();
             VectorQueries = new ChangeTrackingList<VectorQuery>();
         }
-
```

Comment:
Why is this getting removed?

Reply:
Ahh - got it. You need to initialize the QueryCaption and QueryAnswer

Reply:
Would this not automatically get regenerated when you use the autorest.md again? Don't you need to do something special to suppress this from getting generated?

Reply:
We utilize transformations using autorest.md for minor updates, such as renaming and setting types to readonly..etc. However, in this case, where we're introducing a bunch of new types and rearranging properties extensively, using  autorest.md would be very complex. Hence, it is recommended and simpler to use C# customization for incorporating new types and reorganizing properties.

Reply:
To clarify, the generator will automatically suppress the generation of the ctor/method if an identical ctor/method signature already exists in the custom code.